### DISCUSSION TITLE
Updating Chia Nodes Doesn’t cause issues.

### DISCUSSION THREAD

**Discussion by Cminer3207** [swar/Swar-Chia-Plot-Manager#262] [General]
I was trying to say it doesn’t cause a problem, but I totally bolloxed the opening line. 

The problem with using SWAR isn’t updating Plots... it’s updating the YAML without dropping offline and wasting temp-drive write-durability. 

But you guys did a great job explaining WHY the node update isn’t a problem.

**Comment by hateriarch** [comment]
How can that be?  If it is just running the chia command like so

chia plots create -k 32 -b 3300 -t /plot1 -d /chiafarm -r 4 -u 128 -e

and it is all based on your own keys, why would a plot created using SWAR be bad after an update?  

I could just run those commands myself on the command line without using swar and get the same result.

Please explain

**Reply by raid5** [reply]
Agreed, I'm not really understanding what the issue is here.
